R/mobr.R

```diff
@@ -266,7 +275,7 @@ rarefaction = function(x, method, effort=NULL, xy_coords=NULL, latlong=FALSE,
         }
         out = rowSums(1 - p)
     }
-    names(out) = effort
+    names(out) = effort_names
```

Comment:
yes because `effort_names = effort` see where it is first defined at the beginning of the function.